### ISSUE TITLE
[Alterac Valley] Custom implementation of NPC's since Nostalrius! Not Blizzlike!

### ISSUE THREAD

**Issue by Schmoha** [elysium-project/server#1887]
Bug description:
The NPC's in fighting in the middle at the Field of Strife are not supposed to be there. Nostalrius put them in there. Their reasoning is this: https://forum.nostalrius.org/viewtopic.php?f=32&t=35153#p249039

There is no evidence showing the NPC's fighting like that in the middle. No matter what old AV video you watch, you won't find it like on Elysium currently and before on Nostalrius, because this is wrong placement of pre-nerf NPC's and is custom. Nostalrius put it in ther because they didnt know where the pre-nerf NPC's should be.

Please remove them! They dont belong there!

Evidence Video:
Here is a video you can clearly see, there are no NPC's in the middle: https://www.youtube.com/watch?v=uYYYbZ51O8A (Watch 11:30)

Also if you wanted to know whether the AV on Elysium is the same as on Nostalrius, here is a Video of Nostalrius AV: https://www.youtube.com/watch?v=dRg2AUk3COY

If you are interested enought to watch more in that Evidence Video, you can see the pre-nerf NPC's, which never existed on Nostalrius and Elysium.

And by the way, the Evidence Video isnt from patch 1.12.1 or so. It is older than patch 1.8, first look at their gear, second, all the pre-nerf npcs are there, patrolling and defending constructions and beeing placed inside towers and bunkers.

**Comment by thepparker** [comment]
That video is from 2006, which is pointedly POST nerf. You may be correct in that there shouldn't be a bunch of NPCs fighting, but that video doesn't prove it

**Comment by Schmoha** [comment]
Nope, that video is before 2006. It says 2006 in the title, but what does it mean? Did he upload in 2006 or what? You have no idea about the pre nerf NPC's that are shown in the video, moost of them are gone in 2006. In the video they are there and as i said its before patch 1.8. Why do i have to prove what i already said ? I prove it to make the thread look good. You are just clueless about anything right now, why do you even comment ?

**Comment by thepparker** [comment]
So we should take you on your word that it is from a specific point in time? That's not how it works. You are right, I don't know whether it is correct or not, but as I said your 'evidence' isn't really sufficient. 

Also, as a potential developer that would be implementing such a change, you make it pretty unappealing to assist you in rectifying the issue. Perhaps you should try coming back with a modicum of respect or politeness and try again, and someone might be willing to work on the problem (if there is one).

**Comment by Mostprovidedlinks** [comment]
Actually I played in vanilla and there were guards in that area fighting. The date you showed was early 2006 Schmoha. We need to see proof of this in late vanilla

**Comment by Schmoha** [comment]
Its a 30 min video there is a lot to look at, there are actually npcs inside bunkers. no, not bowmen. normal guards

**Comment by Schmoha** [comment]
Also yea if there is one, i should just quit this place, giving elysium to much attention. Go ahead with your custom AV pretending playing vanilla AV, also commanders and lieutents doing to much dmg but yea nobody cares anyway xD. All is in the video, just takes a little of dedication but yea go ahead and try to down this thread or this issue for political correctness, you fools. This bug is not about me or a personal wish, its for the benefit of all, which you dont understand. You dont know whats blizzlike and you dont care, but still dare to come here and say these things.

**Comment by Phoosy** [comment]
You are unusually hostile for someone trying to, apparently attempting to in good faith, report something that may not have been intentional and "Blizzlike" because no one really noticed it before. If you're going to try to argue your point, being an ass and making this some kind of personal crusade for something that is honestly extremely inconsequential to begin with makes it seem like you're just digging for a reason to dislike the project rather than objectively understand that development for an _emulation project_ is a nuanced, complicated process.

If you can find other evidence that further narrows down or defines an AV fight of that area, such as blog posts/guides, videos, or screenshots that pin patches or specific dates better, I guarantee people would be have no problem here - however, the way you've carried yourself now is indicative that you're unable to understand the objection and aren't really willing to even discuss the matter because you're already convinced you're right. If your attitude is derived from this simple of an interaction, to be quite honest the community is probably better off if you weren't here at all.

The issue should remain open if this addition was indeed not intentional, but if no further proof can be found that's more concrete and presented in a less dickish manner then it should be closed pending any updates.

**Comment by Itembugs** [comment]
https://www.youtube.com/watch?v=_WlPYIJV658

there are npcs in that area you mentioned there are none
